frontend/resourceadm/data-mocks/resourceMocks/index.ts

```diff
@@ -0,0 +1,18 @@
+import type { ResourceTypeOption, SupportedLanguageKey } from "app-shared/types/ResourceAdm";
+import { SupportedLanguage } from "resourceadm/types/global";
+
+export const resourcePageTextfieldInputMock1: SupportedLanguageKey<string> = { nb: 'Valid', nn: 'Valid', en: 'Valid' };
+export const resourcePageTextfieldInputMock2: SupportedLanguageKey<string> = undefined;
+export const resourcePageTextfieldInputMock3: SupportedLanguageKey<string> = null;
+
+export const resourceTypeOptionDefaultMock: ResourceTypeOption = 'Default';
+export const resourceTypeOptionIncorrectMock: any = 'Incorrect';
+
+export const nbLanguageMock = 'nb';
```

Comment:
Personally, I would still inline this within the tests instead of referencing a global mock. :)